src/coreclr/vm/jitinterface.h

```diff
@@ -252,7 +252,6 @@ extern "C" FCDECL2(VOID, JIT_WriteBarrier_Callable, Object **dst, Object *ref);
 #define WriteBarrier_Helper JIT_WriteBarrier_Callable
 
 extern "C" FCDECL1(void, JIT_InternalThrow, unsigned exceptNum);
```

Comment:
This helper looks unused too. Can it be deleted as well?

Reply:
I wonder if there's any chance we want to throw exceptions from ASM in the future. I think it's unlikely because ASM would only be used for hot path or special routines (write barrier) that can't be handled by the JIT well. Throws can always be in the caller, like QCall.